src/webui/www/private/confirmdeletion.html

```diff
@@ -14,6 +14,53 @@
             const isDeletingFiles = (new URI().getData('deleteFiles') === "true");
             $('deleteFromDiskCB').checked = isDeletingFiles;
 
+            // Add SVG lock icon to rememberBtn with JS so we can change the fill color
+            new Request({
+                url: '/images/object-locked.svg',
+                method: 'get',
+                onSuccess: function(text, xml) {
+                    // The SVG's height is 32px, which is too big for our dialog
+                    xml.childNodes[0].style.height = '24px';
+                    xml.childNodes[0].style.width = '24px';
+                    $('rememberBtn').appendChild(xml.childNodes[0]);
+                    // Disable button until delete-files checkbox is clicked
+                    disableRememberBtn();
```

Comment:
If possible, I would prefer the icon and its style is declared in html and the initial `disableRememberBtn()` can be called when `domready`.